projects/templates/src/lib/components/po-page-dynamic-table/po-page-dynamic-table.component.ts

```diff
@@ -483,12 +485,13 @@ export class PoPageDynamicTableComponent extends PoPageDynamicListBaseComponent
   }
 
   private loadDataFromAPI() {
-    const { serviceApi, serviceMetadataApi } = this.activatedRoute.snapshot.data;
+    const { serviceApi, serviceMetadataApi, serviceLoadApi } = this.activatedRoute.snapshot.data;
+    const onLoad = serviceLoadApi || this.onLoad;
```

Comment:
Caso for alterar para a sugestão das linhas em branco, como no outro componente, seria interessante alterar aqui também :)

Reply:
Alterado!